tests/Unit/ThirdParty/Plugins/Smush.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace WP_Rocket\Tests\Unit\ThirdParty\Plugins\Smush;
+
+use WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber;
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+
+class Smush extends TestCase
+{
+    /**
+     * Setup constants required by Smush plugin & include the smush.php
+     *
+     * @since 3.4.2
+     * @author Soponar Cristina
+     *
+     * @return void
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if ( ! defined( 'WP_SMUSH_VERSION' ) ) {
+            define( 'WP_SMUSH_VERSION', '3.2.4' );
+        }
+        if ( ! defined( 'WP_SMUSH_PREFIX' ) ) {
+            define( 'WP_SMUSH_PREFIX', 'wp-smush-' );
+        }
+    }
+
+    /**
+     * Test should disable WP Rocket lazy load functionality when Smush lazyload is enabled
+     *
+     * @since 3.4.2
+     * @author Soponar Cristina
+     *
+     */
+    public function testShouldDisableWPRocketLazyLoad()
+    {
+        $subscriber  = new Smush_Subscriber();
+
+        Functions\expect( 'get_option' )
+            ->once() // called once
+            ->andReturn( [ 'lazy_load' => true ] );
+
+        Functions\when( '__' )
+            ->justReturn( 'Smush, ' );
+
+        $this->assertNotEmpty( $subscriber->is_smush_lazyload_active( [] ) );
+    }
+
+    /**
+      * Test should not disable WP Rocket lazy load functionality when Smush lazyload is disabled
+     *
+     * @since 3.4.2
+     * @author Soponar Cristina
+     *
+     */
+    public function testShouldNotDisableWPRocketLazyLoad()
+    {
+        $subscriber  = new Smush_Subscriber();
+
+        Functions\expect( 'get_option' )
+            ->once() // called once
+            ->andReturn( [ ] );
+
+        $this->assertEmpty( $subscriber->is_smush_lazyload_active( [] ) );
+    }
+
+    /**
+     * Test should not disable WP Rocket lazy load functionality when Smush lazyload is disabled
+     *
+     * @since 3.4.2
+     * @author Soponar Cristina
+     *
+     */
+    public function testShouldNotMaybeDeactivateLazyload()
+    {
+        $subscriber  = new Smush_Subscriber();
+
+        Functions\expect( 'get_option' )
+            ->once() // called once
+            ->andReturn();
+
+        $this->assertEmpty( $subscriber->smush_maybe_deactivate_lazyload() );
```

Comment:
the method doesn't return anything, so this assertion is not useful here.

Instead what can be done is to pass Brain Monkey `expect` as tests.

At the start of your class, you need to had the following: `use \Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;`

And in this method, you can do:
```
        Functions\expect( 'update_rocket_option' )
            ->never();
```

I would also add another test to check for the case when we should disable WPR lazyload, using the same logic, but with `update_rocket_option` called once with an argument of `0`:

```
Functions\expect('update_rocket_option')
  ->once()
  ->with('0');
```